src/applications/proxy-rewrite/tests/vrt/teamsite-header-footer.vrt.spec.js

```diff
@@ -30,7 +32,25 @@ describe.each(TEAMSITES)('%s', (sitename, url) => {
   };
 
   beforeAll(async () => {
-    browser = await puppeteer.launch();
+    const options = {
+      uri: ``,
+      json: true,
+      resolveWithFullResponse: true,
+    };
+
+    const test = await http.get({
+      host: 'http://chrome:9222',
```

Comment:
This is still TODO- Once the containers are running, hitting this hostname _should_ work. Puppeteer connects to the headless chrome container through a websocket to do its work.